src/calculateRentalCost.js

```diff
@@ -28,7 +28,19 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const cost = days * 40;
+  const half = 20;
+  const week = 50;
+
+  if (days >= 7) {
+    return cost - week;
+  }
+
+  if (days >= 3) {
+    return cost - half;
+  } else {
```

Comment:
Again, you don't need to use `else` after return statement. If return is performed - code after it won't be executed anyway.